WooCommerce/Classes/POS/Presentation/ItemRowView.swift

```diff
@@ -59,11 +49,34 @@ struct ItemRowView: View {
         .padding(.vertical, Constants.verticalPadding)
         .shadow(color: Color.black.opacity(0.08), radius: 4, y: 2)
     }
+
+    @ViewBuilder
+    private var productImage: some View {
+        if dynamicTypeSize >= .accessibility3 {
+            EmptyView()
+        } else if let imageSource = cartItem.item.productImageSource  {
+            ProductImageThumbnail(productImageURL: URL(string: imageSource),
+                                  productImageSize: Constants.productCardSize,
+                                  scale: scale,
+                                  foregroundColor: .clear)
+            .frame(width: min(Constants.productCardSize * scale, Constants.maximumProductCardSize),
+                   height: Constants.productCardSize * scale)
+            .clipped()
+        } else {
+            // TODO:
+            // Handle what we'll show when there's lack of images:
```

Comment:
I recall we asked at some point if we should re-use the "empty image" asset from WCiOS but we were told it was going to be different, I cannot find the conversation right now but I don't think design followed-up with this item, so we're just showing the grey square as a temporary fallback.

Reply:
It looks like we're keeping it the same way for now p1724121540808449/1724076811.806359-slack-C070SJRA8DP